examples/sinatra/config.ru

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('fishbrain_example', File.dirname(__FILE__))
+
+use OmniAuth::Builder do
+  configure do |config|
```

Comment:
can't this be in the `use OmniAuth::Builder do` in the sinatra app itself?

Reply:
It can, I've added this to the README.md